### DISCUSSION TITLE
How to get connectionId or lastSessionId?

### DISCUSSION THREAD

**Discussion by vforvasile** [TheRealNate/react-native-meteor#88] [Q&A] [answered]
On the web side, we use **Meteor.connection._lastSessionId**.

Any idea on how to get the connectionId with this library?

**Comment by not-ai-yet** [comment, marked as answer]
Hey @vforvasile, this is an interesting use-case. I don't think we have any way to do this currently as we aren't storing the connection id. If you require the connection ID please convert this to an issue and I'll flag it as needs work, otherwise if you'd like to discuss an alternative we can do so.

Thanks!
